CHANGELOG.md

```diff
@@ -1,20 +1,28 @@
 # Release History
 
-## 1.17.0 (TBD)
+## 1.17.0 (2024-05-21)
 
-### New Features
+### Snowpark Python API Updates
+
+#### New Features
 
 - Added support to add a comment on tables and views using functions listed below:
   - `DataFrameWriter.save_as_table`
   - `DataFrame.create_or_replace_view`
   - `DataFrame.create_or_replace_temp_view`
   - `DataFrame.create_or_replace_dynamic_table`
 
-### Improvements
+#### Improvements
 
 - Improved error message to remind users set `{"infer_schema": True}` when reading csv file without specifying its schema.
 
-### Local Testing Updates
+### Snowpark pandas API Updates
+
+#### New Features
+
+- Initial public preview release of Snowpark pandas API.
```

Comment:
nit: Should we maybe add a short description on what the Snowpark pandas API is? Like 1-2 lines max, because it's a changelog.

Reply:
I didn't add more based on what we did for prpr and pupr in both python and pandas API. @sfc-gh-dolee @sfc-gh-mabrennan Let me know if you want to rephrase here.

Reply:
The feature will have a release note in the real Snowflake release notes. I don't think we should add descriptions here. If you want to add a link to the documentation, go ahead.